Examples/Io/Csv/include/ActsExamples/Io/Csv/CsvBFieldWriter.hpp

```diff
@@ -0,0 +1,114 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/MagneticField/InterpolatedBFieldMap.hpp"
+#include "Acts/MagneticField/MagneticFieldProvider.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <array>
+#include <optional>
+
+namespace ActsExamples {
+/**
+ * @brief Writer for B-fields that outputs field data in CSV format.
```

Comment:
Would you mind converting these to `///`?

Reply:
Sure, do you also prefer `//` for non-Doxygen comments?

Reply:
I've gone and converted it all to `///` and `//`.